examples-extra/basic/cli/package.json

```diff
@@ -31,13 +31,14 @@
     "@osdk/api": "workspace:^",
     "@osdk/client": "workspace:^",
     "@osdk/examples.basic.sdk": "workspace:^",
+    "@osdk/foundry": "workspace:^",
     "@osdk/internal.foundry": "workspace:^",
     "pino": "^9.0.0",
+    "pino-pretty": "^11.0.0",
     "tiny-invariant": "^1.3.1"
   },
   "devDependencies": {
     "@osdk/cli": "workspace:^",
-    "@osdk/foundry": "workspace:^",
```

Comment:
Dev dependencies get bundled and that means the symbol we use to make this whole thing work was created twice.